apps/studio/src/components/CmsSidebar/CmsSideNav.tsx

```diff
@@ -18,6 +19,13 @@ import { BiData, BiFile, BiFolder, BiHomeAlt } from "react-icons/bi"
 
 import { trpc } from "~/utils/trpc"
 
+const isAllowedToHaveChildren = (resourceType: ResourceType): boolean => {
+  return (
+    resourceType !== ResourceType.Page &&
+    resourceType !== ResourceType.CollectionPage
```

Comment:
nit: prefer being explicit about what is allowed instead of what is disallowed, prevents subtle bugs for if we ever add other `ResourceType`s

```suggestion
    [ResourceType.RootPage, ResourceType.Collection, ResourceType.Folder].includes(resourceType)
```

Reply:
done downstream using an explicit check!